### DISCUSSION TITLE
版本 4.8.11，组件 tabbar 的placeholder 的作用是什么，为什么fixed 定位之后 placerholder 的位置跟预期不一样。

### DISCUSSION THREAD

**Discussion by Marszht** [youzan/vant#12824] [Q&A]
[issue](issue address](https://github.com/youzan/vant/issues/12823))

**Comment by 1548583993** [comment]
占位。个人认为最佳实践是，在代码层面就将&lt;tabbar/&gt;放在页面对应的位置，即若开启fixed（默认固定于页面底部），就也将&lt;tabbar/&gt;放在代码的最下方（而不是认为既然开启了fixed固定于页面底部，就可以随便放置&lt;tabbar/&gt;）
![111](https://github.com/user-attachments/assets/3ef27402-a4c3-4d45-8326-6afe0f762446)
![2024-11-20_203906](https://github.com/user-attachments/assets/355eb7ea-c606-4573-be32-e179e9cfaf88)
